docs/model/annotations.md

```diff
@@ -0,0 +1,367 @@
+---
+id: annotations
+title: Annotations
```

Comment:
Adding reference?

Reply:
I will merge without, because I think adding "Reference" is a bit redundant as it is self-explanatory. We can always change it if you (or ye) feel strongly about adding it.